css/_vars.scss

```diff
@@ -8,8 +8,8 @@ $color--neutral-bg: #f5f5f5;
 
 /* Popup */
 $popup__width: 270px;
-$popup__height: 505px;
-$popup__height--moz: 520px;
+$popup__height: 540px;
+$popup__height--moz: 540px;
```

Comment:
Notice that originally the heights across browsers were different and now they are the same. Did you verify that the height was correct in FF and Chrome? I would think that FF would be taller.